stdlib/flux_test.go

```diff
@@ -20,28 +20,32 @@ func init() {
-
-	"measurement_tag_keys":   "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
-	"aggregate_window_mean":  "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
-	"aggregate_window_count": "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
-
-	"extract_regexp_findStringIndex": "pandas. map does not correctly handled returned arrays (https://github.com/influxdata/flux/issues/1387)",
-	"partition_strings_splitN":       "pandas. map does not correctly handled returned arrays (https://github.com/influxdata/flux/issues/1387)",
-
-	"http_endpoint": "need ability to test side effects in e2e tests: https://github.com/influxdata/flux/issues/1723)",
+var skip = map[string]map[string]string{
+	"universe": {
+		"string_max":                  "error: invalid use of function: *functions.MaxSelector has no implementation for type string (https://github.com/influxdata/platform/issues/224)",
+		"null_as_value":               "null not supported as value in influxql (https://github.com/influxdata/platform/issues/353)",
+		"string_interp":               "string interpolation not working as expected in flux (https://github.com/influxdata/platform/issues/404)",
+		"to":                          "to functions are not supported in the testing framework (https://github.com/influxdata/flux/issues/77)",
+		"covariance_missing_column_1": "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
+		"covariance_missing_column_2": "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
+		"drop_before_rename":          "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
+		"drop_referenced":             "need to support known errors in new test framework (https://github.com/influxdata/flux/issues/536)",
+		"yield":                       "yield requires special test case (https://github.com/influxdata/flux/issues/535)",
+		"task_per_line":               "join produces inconsistent/racy results when table schemas do not match (https://github.com/influxdata/flux/issues/855)",
+		"integral_columns":            "aggregates changed to operate on just a single columnm.",
+	},
+	"http": {
+		"http_endpoint": "need ability to test side effects in e2e tests: https://github.com/influxdata/flux/issues/1723)",
+	},
+	"testing/chronograf": {
+		"measurement_tag_keys":   "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
+		"aggregate_window_mean":  "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
+		"aggregate_window_count": "unskip chronograf flux tests once filter is refactored (https://github.com/influxdata/flux/issues/1289)",
+	},
+	"testing/pandas": {
```

Comment:
Should these tests be in the `strings` package?

Reply:
They seemed to be directly related to some tests involving pandas. I don't really know enough about them though to know whether they should be moved. I mostly just tried to move `testdata` to `universe` and then break up anything in there. I've mostly left anything else unmoved, but this will open things up to us moving them if it is more appropriate.

Reply:
Ok to me it seems like they belong with the `strings` package. But I don't really know enough about them either.